internal/pkg/consensus/election.go

```diff
@@ -5,7 +5,6 @@ import (
 	"math/big"
 
 	address "github.com/filecoin-project/go-address"
-	"github.com/filecoin-project/sector-storage/ffiwrapper"
```

Comment:
This makes me think it will make more sense to merge #4005 first to avoid unnecessary churn.  After that lands you shouldn't need to modify this file at all to get this compiling.